.github/workflows/ci.yml

```diff
@@ -0,0 +1,12 @@
+name: Community website CI
+
+on:
+  pull_request:
```

Comment:
yes. forgot to mention. seems like a good idea to run on PRs to me. would you mind sending a PR to my fork's main branch?

Reply:
Pushed one yesterday, did it work?

Reply:
I guess it's a little unnecessary to have the nikola-build plus the RTD check. But I think we can add linters and formatters and the nikola-build check could be easier for spotting build issues, rather than having to go to RTD and read through their log files.

Reply:
Sure, I think it's fine :+1: